backend/complexity_analyser/src/main/scala/complexity_analyser/HaskellProcessor.scala

```diff
@@ -180,27 +177,22 @@ class HaskellProcessor(modelAnswer: File, studentAnswer: File) {
   def calculateScore(deltas: ArrayBuffer[(String, Double)]) = {
     var score = 100
     val annotations = new ArrayBuffer[Error]
-    for ((n, v) <- deltas) {
-      val diff = Math.abs(v).round
-      if (diff > 50) {
-        if (score > 0) {
-          score -= (diff / 8).toInt
-        }
+    var eff = ""
+    for ((n, diff) <- deltas) {
+      if (Math.abs(diff) > 25000) {
+        score -= (diff / 20000).toInt
```

Comment:
Maybe a little explanation on the numbers? I assume this is some runtime constant threshold but a comment explaining or a constant var name would be nice